.changeset/chatty-shoes-hunt.md

```diff
@@ -0,0 +1,9 @@
+---
+'@graphql-eslint/eslint-plugin': major
+---
+
+feat: split `recommended` config to two configs `schema-recommended` and `operations-recommended`
+
+`recommended` and `all` configs were divided to 4 configs `schema-recommended`
+, `operations-recommended`, `schema-all` and `operations-all`. This was done in order to
+use `recommended` and `all` configs in `schema-only` projects where it is not possible to provide operations.
```

Comment:
Per @B2o5T's feedback [here](https://github.com/dotansimha/graphql-eslint/pull/752#discussion_r744166414), `schema-all` is intended to assign all schema rules regardless of requirement for `parserOptions.operations`.

Reply:
Hi @JustinTRoss, I changed this after your feedback, so now schema-only projects will be able to use schema-all config :)